woocommerce/woocommerce
# WooCommerce Docs: Handle multiple category posts

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





When creating new content to work on the post deletion issue, I noticed only one Category post was being added to the manifest. 

This PR fixes a scenario where `category.posts` was getting recreated as an empty array with each loop, erasing any previous posts.

Note the side effect of this is that the manifest now has a top level `posts: []` and I began the process of using the `checkReadme` bool to determine if the iteration was the top level to avoid adding `posts: []`. But the more I think about it, thats ok and even though we're not likely to use it, it should still be there.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Note that a new markdown file was added in the `testing` category.
2. Run the command to generate a manifest
```
pnpm utils md-docs create ./plugins/woocommerce-docs/example-docs woodocs --outputFilePath ./plugins/woocommerce-docs/scripts/manifest.json
```
3. Note the testing category now has two entries. 
4. `pnpm test`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
